src/Xamarin.Android.Build.Tasks/Xamarin.Android.Designer.targets

```diff
@@ -0,0 +1,121 @@
+<!--
```

Comment:
Would it be possible to place this file into the `src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android` directory?

Additionally, as this is a new file we should reformat this file to conform with our current indentation conventions.